Arrowgene.Ddon.GameServer/Handler/CharacterCommunityCharacterStatusGetHandler.cs

```diff
@@ -15,11 +18,40 @@ public CharacterCommunityCharacterStatusGetHandler(DdonGameServer server) : base
         {
         }
 
-        public override PacketId Id => PacketId.C2S_CHARACTER_COMMUNITY_CHARACTER_STATUS_GET_REQ;
+        // public override PacketId Id => PacketId.C2S_CHARACTER_COMMUNITY_CHARACTER_STATUS_GET_REQ;
 
-        public override void Handle(GameClient client, IPacket packet)
+        public override void Handle(GameClient client, StructurePacket<C2SCharacterCommunityCharacterStatusGetReq> packet)
         {
-            client.Send(InGameDump.Dump_65);
+            // client.Send(InGameDump.Dump_65);
+
+            List<CDataCharacterListElement> updateCharacterList = new List<CDataCharacterListElement>();
+            List<CDataUpdateMatchingProfileInfo> updateMatchingProfileList = new List<CDataUpdateMatchingProfileInfo>();
+            
+            List<ContactListEntity> friends = Database.SelectContactsByCharacterId(client.Character.CharacterId);
+            
+            foreach (var f in friends)
+            {
+                if (f.Type != ContactListType.FriendList || f.Status != ContactListStatus.Accepted) continue;
```

Comment:
I'm not a fan of these kind of unbracketed if statements. I've seen a lot of bugs where some other person comes to change and then made a mistake due to this kind of style. If Laith is ok with this though, then it is fine.